modules/apps/organizations/organizations-test/src/testIntegration/java/com/liferay/organizations/service/test/OrganizationLocalServiceWhenSearchingOrganizationsTreeTest.java

```diff
@@ -182,29 +181,22 @@ protected String toString(Organization organization) {
 		map.remove("createDate");
 		map.remove("modifiedDate");
 
-		Stream<Map.Entry<String, Function<Organization, Object>>> stream =
-			SearchStreamUtil.stream(map.entrySet());
+		Map<String, String> organizationAttributesMap = new HashMap<>();
```

Comment:
@julschong You can simplify this method with `Organization.getModelAttributes()`.